ratbagd/ratbagd.c

```diff
@@ -204,6 +204,67 @@ static const sd_bus_vtable ratbagd_vtable[] = {
 	SD_BUS_VTABLE_END,
 };
 
+/* 32 pre-picked names. If you have more than 32 devices connected and need
+ * ratbag to control them all, congratulations, you are official a niche
+ * use-case.
+ */
```

Comment:
in that case, we should add a google translate API and convert the rodents names to Swedish or Icelandic :)

Reply:
urgh. UTF-8 is already a pain without å and the like :)